styles.css

```diff
@@ -0,0 +1,344 @@
+  opacity: 1;
+}
+
+.main {
+  padding: 195px 66% 245px 0;
+  background: center right no-repeat url(img/Frame.svg);
+}
+
+.main__title {
+  margin: 0 0 40px 0;
+  font: bold 66px/87px 'PT Serif', serif;
+}
+
+.main__subtitle {
+  margin: 0 0 41px 0;
+  width: 455px;
+  font: 22px/25px Roboto, arial, helvetica, sans-serif;
+}
+
+.main__title-text_highlight {
+  background: #ecf3ff;
+}
+
+.main__button {
+  display: inline-block;
+  font: 18px/21px Roboto, arial, helvetica, sans-serif;
+  color: #fff;
+  background: #0056ec;
+  padding: 18px 30px;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity .5s;
+}
+
+.main__button:hover {
+  opacity: .5;
+}
+
+@media (max-width: 1366px) {
+  .wrapper {
+    margin: auto;
+    width: 69%;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .logo {
+    display: inline-block;
+    position: relative;
+    padding: 40px 0;
+    padding-left: 44px;
+    font: bold 24px/31px 'PT Serif', serif;
+  }
+
+  .menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .menu__item {
+    position: relative;
+    margin-right: 27px;
+    font: 500 14px/16px Roboto, arial, helvetica, sans-serif;
+    color: #000;
+    text-decoration: none;
+    transition: opacity .5s;
+  }
+
+  .main {
+    padding: 93px 58% 120px 0;
+    background: center right no-repeat url(img/Frame.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0 0 40px 0;
+    width: 434px;
+    height: 195;
+    font: bold 50px/66px 'PT Serif', serif;
+  }
+
+  .main__subtitle {
+    margin: 0 0 41px 0;
+    width: 367px;
+    height: 43px;
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    font: 14px/16px Roboto, arial, helvetica, sans-serif;
+    padding: 14px 16px;
+  }
+}
+
+@media (max-width: 1170px) {
+  .wrapper {
+    width: 81%;
+    margin: 0 47px 0 39px;
+  }
+
+  .main {
+    padding: 94px 58% 120px 0;
+    background: center right no-repeat url(img/Frame.svg);
+    background-size: 55%;
+  }
+}
+
+@media (max-width: 1024px) {
+  .wrapper {
+    width: 88%;
+  }
+
+  .main {
+    padding: 94px 58% 120px 0;
+    background: center right no-repeat url(img/Frame.svg);
+    background-size: 55%;
+  }
+}
+
+@media (max-width: 940px) {
+  .wrapper {
+    width: 95%;
+  }
+
+  .main {
+    padding: 94px 55% 120px 0;
+    background: center right no-repeat url(img/Frame.svg);
+    background-size: 53%;
+  }
+}
+
+@media (max-width: 768px) {
+  .wrapper {
+    width: 85%;
+  }
+
+  .header {
+    margin-bottom: 28px;
+  }
+  .menu {
+  display: none;
+}
+
+.logo {
+  visibility: hidden;
+  padding: 53px 0 0 0;
+}
+
+.logo__circle_color_blue {
+  visibility: visible;
+}
+
+.logo__cirle_color_dark-blue {
+  visibility: visible;
+}
+
+.burger {
+  visibility: visible;
+}
+
+.main {
+  text-align: center;
+  padding: 74% 0  0;
+  background-size: 74%;
+  background-position: top center;
+}
+
+    .main__title {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/site99/pullRequest?prid=6205931)